dask/array/tests/test_fft.py

```diff
@@ -18,6 +18,20 @@ def key(k):
     return sorted(a.dask, key=key) == sorted(b.dask, key=key)
 
 
+@pytest.fixture(
+    params=[
+        "fft",
+        "ifft",
+        "rfft",
+        "irfft",
+        "hfft",
+        "ihfft",
+    ]
+)
+def funcname(request):
+    return request.param
```

Comment:
...but then we need `for`-loops everywhere. IMHO either we should use the `fixture` or we should revert the last change.